sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2291,6 +2311,13 @@ class SQLConf extends Serializable with Logging {
   def wholeStageSplitConsumeFuncByOperator: Boolean =
     getConf(WHOLESTAGE_SPLIT_CONSUME_FUNC_BY_OPERATOR)
 
+  def extendedEventInfo: Boolean =
+    getConf(EXTENDED_EVENT_INFO)
```

Comment:
to be consistent style, no need to merge above 2 lines to 1 line

Reply:
Done.